modules/samples-operator-crd.adoc

```diff
@@ -4,25 +4,25 @@
 
 
 [id="samples-operator-crd{context}"]
-= Accessing the Samples Operator file
+= Accessing the Samples Operator configuration
 
 You can configure the Samples Operator by editing the file with the provided
 parameters.
 
 .Procedure
 
-*  Access the Samples Operator file:
+*  Access the Samples Operator configuration:
 +
 ----
-# oc get configs.samples.operator.openshift.io -n openshift-cluster-samples-operator
+$ oc get configs.samples.operator.openshift.io/cluster -o yaml
 ----
 +
-The following is an example of the Samples Operator file:
+The following is an example of the Samples Operator configuration:
```

Comment:
You need a noun after "following," but I'd probably say "The Samples Operator configuration resembles the following example:" instead.

> :no_entry_sign: following n
Do not use as a noun except to refer to a group of followers, adherents, or partisans. For example, do not
write “Do the following” or “The functions include the following” as lead-in wording to a list. Use
“following” only as an adjective, as in “Complete the following steps” or “The adapter has the following
functions.”